### ISSUE TITLE
No supporting technology for WCAG success criterion 1.3.4

### ISSUE THREAD

**Issue by Wildebrew** [w3c/wcag21#723]
I really like the idea of identifying the purpose of common controls.
But I have a two-fold problem with making 
[Success criterion 1.3.4](http://www.w3.org/TR/WCAG21/#identify-common-purpose)
part of the WCAG 2.1 recommendation.

1. There are too many common purposes listed. There are a total of 64 [common and common input purposes](http://www.w3.org/TR/WCAG21/#control-purposes) listed. How can users learn to identify so many common purposes? If the list were trimmed by 50 to 75% maybe it would be easier to work with. Also my experience working with a lot of different web professionals is that each will have a slightly different interpretation of these concepts, and we will end up with a lot of inconsistencies that greatly reduces the benefits. 
2. (more importantly) there is no existing HTmL or ARIA markup that lets webpage authors implement this success criterion, at least as far as I know. Something could be done with data attributes.

A documented technical solution, complete with examples, should be in place before authors are expected to comply with this requirement.

I find this one of the most interesting and forward-looking success criteria, and I am 100% behind the concept, but I feel better technology support and documentation needs to be in place before this can be made a conformance requirement.

**Comment by awkawk** [comment]
Thank you for your comment. The deadline for comments has passed, but the Working Group will make efforts to consider your comment and any suggestions for improvements. (Birkir, I'm just adding this to each issue that comes in now, I know that you understand, but we are happy to receive your thoughts and will be trying to take them into account)

**Comment by awkawk** [comment]
(Official WG Response)
Thank you for your comment. The updated SC now addresses both of your concerns as the list of link and button purposes has been removed and the input purposes are based on the HTML 5.2 autofill values.
